### PR TITLE
Sam bug fixes

### PR DESCRIPTION
**Pull request by sam-warren**
Several bug fixes and improvements. 
- Added missing form fields to 24h / VI captured in DF-2714
- DF-2740
- DF-2724
- DF-2639
- DF-2531 (this is a duplicate isue of DF-2714)
- DF-2665
- DF-2666
- DF-2730

### CODE REVIEW

File: python/prohibition_web_svc/middleware/event_middleware.py [hunk 1]
```diff
@@ -83,28 +83,32 @@ def save_event_data(**kwargs) -> tuple:
...
+            confirmation_of_service_date=data.get(
+                'confirmation_of_service_date'),
             agency_file_no=data.get('agency_file_no'),
             created_dt=date_created,
             updated_dt=date_created,
             created_by=user_guid,
             updated_by=user_guid,
         )
         if data.get('VI'):
+            # TODO: ADD REGIST. OWNER EMAIL
```
Comment (JustinMacaulay) [inline python/prohibition_web_svc/middleware/event_middleware.py:108]: Is this still needed?

File: python/prohibition_web_svc/models.py [hunk 2]
```diff
@@ -577,12 +580,14 @@ class VIForm(db.Model):
...
     event_id = db.Column(db.Integer, db.ForeignKey('event.event_id'))
     gender = db.Column(db.String)
+    driver_is_regist_owner = db.Column(db.String)
     driver_licence_expiry = db.Column(db.DateTime)
     driver_licence_class = db.Column(db.String)
     unlicenced_prohibition_number = db.Column(db.String)
     belief_driver_bc_resident = db.Column(db.String)
     out_of_province_dl = db.Column(db.String)
     out_of_province_dl_number = db.Column(db.String)
+    out_of_province_dl_expiry = db.Column(db.String)
```
Comment (JustinMacaulay) [inline python/prohibition_web_svc/models.py:590]: was this column added to the dataclasses up top?

File: roadside-forms-frontend/frontend_web_app/src/components/CommonForm/registeredOwnerInfo.js [hunk 3]
```diff
@@ -1,89 +1,177 @@
...
-          "regist_owner_city": values['driver_city'],
-          "regist_owner_prov_state":  values['driver_prov_state'],
-          "regist_owner_postal":  values['driver_postal'],
-        };
-        setValues({
-            ...values,
-            ...driverInfo,
-          });
-        };
+  // TODO: Clear driver first + last name + DOB when corporate ownership checked
```
Comment (JustinMacaulay) [inline roadside-forms-frontend/frontend_web_app/src/components/CommonForm/registeredOwnerInfo.js:35]: just checking again if this is still needed

File: roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/policeDetails.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ export const PoliceDetails = (props) => {
               <Row>
                 <Col>
                   <Radio
-                    label="If yes, what test was administered?"
+                    label="If yes, what test  administered?"
```
Comment (JustinMacaulay) [inline roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/policeDetails.js:36]: there's an extra space here

**Review by JustinMacaulay — COMMENTED** [review]
Just a few comments.

File: python/prohibition_web_svc/middleware/event_middleware.py (hunk 1 above)
Comment (sam-warren) [reply python/prohibition_web_svc/middleware/event_middleware.py:108]: Removed

File: roadside-forms-frontend/frontend_web_app/src/components/CommonForm/registeredOwnerInfo.js (hunk 3 above)
Comment (sam-warren) [reply roadside-forms-frontend/frontend_web_app/src/components/CommonForm/registeredOwnerInfo.js:35]: removed

File: roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/policeDetails.js (hunk 4 above)
Comment (sam-warren) [reply roadside-forms-frontend/frontend_web_app/src/components/Forms/TwentyFourHourForm/policeDetails.js:36]: fixed

File: python/prohibition_web_svc/models.py (hunk 2 above)
Comment (sam-warren) [reply python/prohibition_web_svc/models.py:590]: added

**Review by JustinMacaulay — APPROVED** [review]
Looks good

**Pull request merged**